### ISSUE TITLE
[ FF7 ]"Filled" bar UI graphics are upside-down

### ISSUE THREAD

**Issue by HeyItsLollie** [julianxhokaxhiu/FFNx#611]
This one is just funny: The graphics used for "filled" bars have been upside down this entire time.

![Bars](https://github.com/julianxhokaxhiu/FFNx/assets/30498581/5a08b287-f27b-4a2b-b7f3-753892ef7306)

**Comment by TailsGuy** [comment]
Actually, I think that's just a design difference, the resolution of PSX is different than PC, that's why the font and menus are different. As you can see, there's a lot more space for the "M" and "E" to be defined in the PC version, and the numbers are much different and less SNES-esque like the original. So I wouldn't say that it's upside down, it's just making use of the 480p resolution that PC had.
Now, I'm actually an enjoyer of the more pixelated PSX style, but I also appreciate PC just being more of a upscaled version of PSX while retaining its style in UI and font [for the most part]
There's also an interesting difference between the Japanese version and the Western releases, the battle UI is different.

**Comment by HeyItsLollie** [comment]
No, it's really just upside-down. Here's a comparison where I've isolated the filled bars by removing their frames.

![BarCompare](https://github.com/julianxhokaxhiu/FFNx/assets/30498581/a1f205cc-6798-4a14-9d87-d32cd53940ad)

I actually tried tracking down the graphics for these bars, but I could not turn them up. It was only after doing these comparisons that I realized... the bars are achieved using two quads with vertex colors. So this can't be fixed by simply flipping images — This probably requires a code fix, or possibly even model fixes if they aren't just generated at runtime.
